src/pages/Token/index.tsx

```diff
@@ -38,12 +38,16 @@ const TokenPage = () => {
   const fetchToken = useCallback(async () => {
     if (!api?.nft || !currentChainId) return;
     setIsFetchingToken(true);
-    const token = await api.nft?.getToken(Number(collectionId), Number(id));
-    setIsBundle(await api.nft?.isBundle(Number(collectionId), Number(id)));
-
-    const collection = await getCollection(Number(collectionId));
-    if (token) setToken(token);
-    if (collection) setCollection(collection);
+    try {
```

Comment:
what is this try/catch for? Neither our getToken/isBundle/getCollection throws error neither sdk's methods